### ISSUE TITLE
SF_ACC: [ui5-shellbar] visual focus outline missing in shellbar menu & profile menu options on hover

### ISSUE THREAD

**Issue by pankajpandey01** [SAP/ui5-webcomponents#5830]
### **Bug Description**
[ui5-shellbar] visual focus outline missing in shellbar menu & profile menu options on hover

### **Expected Behavior**
visual focus should be present a in below UI5 example:
https://openui5.hana.ondemand.com/entity/sap.f.ShellBar/sample/sap.f.sample.ShellBar

### **Steps to Reproduce**
1. Go to 'https://sap.github.io/ui5-webcomponents/playground/components/ShellBar/'
2. Hover on shellbar menu and profile menu options

### **Context**
- UI5 Web Components version: {1.7.0}
- OS/Platform: {All}
- Browser: {All}
- Affected component: {ui5-shellbar}

### **Log Output / Stack Trace / Screenshots**
<img width="316" alt="Screen Shot 2022-09-19 at 1 55 27 PM" src="https://user-images.githubusercontent.com/67934622/191114838-9193661d-4dff-48a1-8336-9be996cae644.png">
<img width="623" alt="Screen Shot 2022-09-19 at 1 55 19 PM" src="https://user-images.githubusercontent.com/67934622/191114856-d6f5cfd1-201f-4d18-ac6a-df7e7f8a3cd6.png">

### **Priority**
- [ ] Low
- [x] Medium
- [ ] High
- [ ] Very High

The priority indicates the severity of the issue. To set the appropriate priority consider the following criteria:
- Accessibility issue - Medium

**Note:** The priority might be re-evaluated by the issue processor.

### **Stakeholder Info** (if applicable)

- Organization: {SuccessFactors}
- Business impact: {Accessibility issue}

**Comment by plamenivanov91** [comment]
Hello @SAP/ui5-webcomponents-topic-p ,

The reporter's problem is that when we hover on an item from the drop-down menu of the ShellBar, we have only hover effect (darker background), but the focus isn't changing on the currently hovered item, where is in the OpenUI5 the focus changes on the hovered item.

It's currently unclear whether or not this is the correct behavior.

Please follow up on this issue.

Regards,
Plamen Ivanov

**Comment by dobrinyonkov** [comment]
Hello, we discussed with our UX team that the expected behavior is a wrong implementation. The focus shouldn’t move on mouse hover. So the way it is implemented [here is correct](https://sap.github.io/ui5-webcomponents/playground/components/ShellBar/)
